### PR TITLE
gnrc: remove unnecessary (and possibly dangerous) packed attributes 

### PR DESCRIPTION
**Pull request by authmillenon**
Does what it says on the tin.

### CODE REVIEW

**Comment by haukepetersen** [comment]
ACK

File: sys/include/net/ng_ipv6/addr.h [hunk 1]
```diff
@@ -49,7 +49,7 @@ extern "C" {
 /**
  * @brief Data type to represent an IPv6 address.
  */
-typedef union __attribute__((packed)) {
+typedef union {
     uint8_t u8[16];             /**< devided by 16 8-bit words. */
```
Comment (gebart) [inline sys/include/net/ng_ipv6/addr.h]: could you correct the speling errors while you're touching this file?
s/devided/divided/g

**Comment by authmillenon** [comment]
Addressed @gebart's comment

**Comment by authmillenon** [comment]
(@gebart: please re-ACK)

**Comment by gebart** [comment]
ACK

**Comment by PeterKietzmann** [comment]
Two ACKs are enough. And go

**Comment by OlegHahm** [comment]
:+1: for the change

**Pull request merged**